nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
I think the external clients should not worry about getting the connection and returning it since I already included `connect()` and `close()` before and after querying respectively.
They just need to call `query()` everytime. Internally, `query()` will get a connection from the pool, execute the query, and then return the connection to the pool.